images/Dockerfile.debian

```diff
@@ -11,7 +11,7 @@ ARG BASE_IMAGE=debian:testing
 # we use testing for amd64 because zfsutils-linux is not available in bookworm
 FROM $BASE_IMAGE AS amd64-upstream
 RUN <<EOF
-echo "deb http://deb.debian.org/debian bookworm main contrib" >> /etc/apt/sources.list.d/kairos.list
+echo "deb https://ftp.debian.org/debian bookworm main contrib" >> /etc/apt/sources.list.d/kairos.list
```

Comment:
I wonder what is better long term, since this will stay as part of the configuration of the debian images released

Reply:
It will be there but since the system can't be upgraded and packages can't be installed (read-only fs) it doesn't matter. It's only used once when building the image.